lib/entities/quote/ClassicQuote.ts

```diff
@@ -205,11 +206,13 @@ export class ClassicQuote implements IQuote {
     return this.request.info.slippageTolerance ? parseFloat(this.request.info.slippageTolerance) : -1;
   }
 
-  public getPortionBips(): number | undefined {
-    return this.quoteData.portionBips;
-  }
+  public get portion(): Portion | undefined {
+    if (this.quoteData.portionBips === undefined || this.quoteData.portionRecipient === undefined) return undefined;
 
-  public getPortionRecipient(): string | undefined {
-    return this.quoteData.portionRecipient;
+    return {
+      bips: this.quoteData.portionBips,
+      recipient: this.quoteData.portionRecipient,
+      type: PortionType.Flat,
```

Comment:
@unicorn-here whats the plan w/ portiontype? seems its always flat, and not passed in or fetched from anywhere currently

Reply:
ah i see, it's interesting that the portion type is given in the URA request but not returned in the routing api request. I'll merge it in

Reply:
I guess one question is why do we receive portion in the request but also from routing API?

Reply:
ah, should we be asserting then that the values returned from routing api === the value we had originally?

Reply:
also related, why do we use the classic portion in https://github.com/Uniswap/unified-routing-api/blob/1c1933e2393d83a0cd95f69779d92f7c5355a58a/lib/entities/quote/DutchQuote.ts#L206-L207? Do we expect it to ever change from what the Dutch quote originally had?